grand_challenge_forge/partials/example-evaluation-method/helpers.py

```diff
@@ -8,6 +9,17 @@
 
 import psutil
 
+logger = logging.getLogger("helpers")
```

Comment:
This is more conventional:

```suggestion
logger = logging.getLogger(__name__)
```

Reply:
It is! I've changed it now but had it to the same style as in evaluate.py. Since there, it would print as ("__main__") and imho that is a bit confusing. So I'll keep it explicitly set as "evaluate" there.